lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -2,6 +2,14 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
 
+    expose :resize
+    expose :confirm_resize
```

Comment:
The method exposed is `resize_confirm` [providers/cloud_manager/vm.rb#L49](https://github.com/ManageIQ/manageiq/blob/33d0fd7d4b5093599825b628728bc7bdde15c81e/app/models/manageiq/providers/cloud_manager/vm.rb#L49)

Does this method return an object?